shared/js/ui/models/whitelist.es6.js

```diff
@@ -44,9 +50,27 @@ Whitelist.prototype = window.$.extend({},
           value: true
         }
         })
+
+        this.setWhitelistFromSettings()
       }
 
       return domain
+    },
+
+    setWhitelistFromSettings: function () {
+      let self = this
+      this.fetch({getSetting: {name: 'whitelisted'}}).then((list) => {
+        list = list || {}
+        let wlist = Object.keys(list)
+        wlist.sort()
+
+        // Prevent sealing array elements
+        self.list = [...wlist]
+
+        // Publish whitelist change notification via the store
+        // used to know when to rerender the view
+        self.set('whitelist', wlist)
```

Comment:
Having two properties seems superfluous. I think you can just do `self.set('list', wlist)` and listen for that property on the view?

Reply:
We talked about this but I'll just write it down here as well: sending the property that's actually being changed in the model doesn't actually update the view, for currently unknown reasons.
We agreed on not merging this and taking more time to understand why this happens and how to fix it, to avoid piling technical debt that will just make our lives harder later on.